scripts/update_sha.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import sys
+import json
+
+
+def main(filename, sha, json_file):
+    with open(json_file, "r") as f:
+        data = json.load(f)
+    if 'sha256sums' not in data:
```

Comment:
nit: `data.setdefault('sha256sums', dict())[filename] = sha` to not repeat the key string many times.